src/test/scala/com/amazon/deequ/analyzers/jdbc/JdbcFixtureSupport.scala

```diff
@@ -5,6 +5,28 @@ import scala.collection.mutable
 
 trait JdbcFixtureSupport {
 
+  def hasTable(conn: Connection, tableName: String): Boolean = {
```

Comment:
This method should not be in fixture support, because it does not return a fixture. Perhaps move it into the test suite or create a new test suite and move it there

Reply:
Name JdbcFixtureSupport does not imply that it only contains fixtures